cmd/podman/common.go

```diff
@@ -500,3 +500,22 @@ func scrubServer(server string) string {
 	server = strings.TrimPrefix(server, "https://")
 	return strings.TrimPrefix(server, "http://")
 }
+
+// UsageTemplate returns the usage template for podman commands
+// This blocks the desplaying of the global options. The main podman
+// command should not use this.
+func UsageTemplate() string {
+	return `Usage:{{if .Runnable}}
+  {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
+  {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+Aliases:
+  {{.NameAndAliases}}{{end}}{{if .HasExample}}
+Examples:
+{{.Example}}{{end}}{{if .HasAvailableSubCommands}}
+Available Commands:{{range .Commands}}{{if (or .IsAvailableCommand (eq .Name "help"))}}
+  {{rpad .Name .NamePadding }} {{.Short}}{{end}}{{end}}{{end}}{{if .HasAvailableLocalFlags}}
+Flags:
+{{.LocalFlags.FlagUsages | trimTrailingWhitespaces}}{{end}}{{if .HasAvailableInheritedFlags}}
+{{end}}
+`
+}
```

Comment:
Can you add some blank lines before each heading such as "Aliases:", "Examples:", "Flags:" and others?  Otherwise they're all smashed together.

```
# podman top --help
Display the running processes of the container.  Specify format descriptors
to alter the output.  You may run "podman top -l pid pcpu seccomp" to print
the process ID, the CPU percentage and the seccomp mode of each process of
the latest container.

Format Descriptors:
args,capamb,capbnd,capeff,capinh,capprm,comm,etime,group,hgroup,hpid,huser,label,nice,pcpu,pgid,pid,ppid,rgroup,ruser,seccomp,state,time,tty,user,vsz

Usage:
  podman top [flags]
Examples:
CONTAINER-NAME [format descriptors]
Flags:
  -h, --help     help for top
  -l, --latest   Act on the latest container podman is aware of
```